examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -5341,7 +5341,7 @@ endpoint 0 {
     callback attribute eventList;
     callback attribute attributeList;
     ram      attribute featureMap default = 2;
-    ram      attribute clusterRevision default = 1;
+    ram      attribute clusterRevision default = 2;
```

Comment:
I am not sure we can do this. This seems to be set for wifi/thread and in that case revision 2 needs SupportedWifiBands or SupportedThreadFeatures/ThreadVersion